src/Infrastructure.EntityFramework/NotificationCenter/Configurations/NotificationStatusEntityTypeConfiguration.cs

```diff
@@ -10,9 +10,13 @@ public class NotificationStatusEntityTypeConfiguration : IEntityTypeConfiguratio
     public void Configure(EntityTypeBuilder<NotificationStatus> builder)
     {
         builder
-            .HasKey(ns => new { ns.UserId, ns.NotificationId })
+            .HasKey(ns => new { ns.NotificationId, ns.UserId })
```

Comment:
I know ordering can have performance implications with some database providers. 

In the main query, we join on both properties but have `UserId` in the Where clause.

We should probably test which way is more performant or if it affects performance at all before making this change in light of MySQL migrations breaking.

It might make sense to have it be its own separate ticket and scoping it out of this one.

Reply:
I will revert this for now. My worry is that in EF we have `UserId, NotificationId` as columns order, while SQL Server have it as `NotificationId, UserId`, so it's a bit inconsistent. Also a secondary (non-primary) index is needed to match on the second column (i.e. delete by UserId only or join on NotificationId only for deletes).

Reply:
Fixed in 1c48155